test/functional/tests/incremental_load/test_core_pool.py

```diff
@@ -0,0 +1,122 @@
+#
+# Copyright(c) 2019 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+import pytest
+
+from api.cas import casadm
+from api.cas.core import CoreStatus
+from core.test_run import TestRun
+from storage_devices.disk import DiskTypeSet, DiskType, DiskTypeLowerThan
+from test_tools.disk_utils import Filesystem
+from test_utils.output import CmdException
+from test_utils.size import Size, Unit
+
+
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+def test_attach_core_pool():
+    """
+    title: Attaching from core pool on cache load.
+    description: |
+      Check that CAS has the ability on cache load to attach core devices that were added to
+      core device pool if those devices were previously used by cache instance being loaded.
+      Prevent attaching core device if they were not previously used.
+    pass_criteria:
+      - No system crash while reloading CAS modules.
+      - Core device was added successfully to core pool.
+      - Core device has been successfully attached to cache on cache load.
+      - Second core device was not attached to the cache instance.
+    """
+    with TestRun.step("Prepare devices."):
+        cache_disk = TestRun.disks["cache"]
+        cache_disk.create_partitions([Size(2, Unit.GibiByte)])
```

Comment:
Is it intended that cache dev is bigger than core?

Reply:
Actually it doesn't make a difference in those tests if core or cache is bigger than the other one.